src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,30 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, setActiveTabId }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={cn({ 'is-active': selectedTab.id === tab.id })}
+              data-cy="Tab"
+            >
+              <a
+                onClick={() => setActiveTabId(tab.id)}
```

Comment:
don't forget to add the check if this tab is not already selected
<img width="618" alt="image" src="https://github.com/mate-academy/react_tabs-js/assets/70220667/622a7ca4-643e-4e7c-a1f0-3ba5e08f9cc2">